### DISCUSSION TITLE
IOS version

### DISCUSSION THREAD

**Discussion by RevanderKh** [stoatchat/discussions#804] [Feature Suggestions]
Is it worth waiting for the iOS version of the app sometime? Because creating a shortcut to the home screen in the Safari browser is not very comfortable

**Comment by IAmTomahawkx** [comment]
You can try the iOS version here: https://github.com/revoltchat/ios

**Reply by BoardingRaichu** [reply]
Can you explain how this works? This seems very technical and I don’t know how this works. I’m not seeing a download link for an iOS version here, just a bunch of files that I’m unsure what to do with. I’m on an iPad right now.

**Comment by RevanderKh** [comment]
What should I do if I don't have a Mac OS device? Windows and iOS only

**Comment by Feldherren** [comment]
Is the iOS version still in development? I know people who aren't interested in switching to Stoat unless the iOS mobile version is workable.

**Reply by Anvnamnet** [reply]
It is. And currently not working in Testflight.
So no iOS yet.

Here's the Github for iOS: https://github.com/stoatchat/for-ios

**Comment by justyrofoam** [comment]
Stoat is unusable on a phone at the moment since the interface is not responsive/optimized for mobile. I can get by with having it on the computer only, but I know many friends who won’t.

**Reply by Anvnamnet** [reply]
Try https://old.stoat.chat 

Heard it works better in iOS browsers.

**Reply by justyrofoam** [reply]
I would say it is definitely more functional, but it would be great if the default experience was also mobile-optimized as I manage projects on Discord (which I’m migrating to Stoat) and I need to be able to react quickly at times. I’m ok with not having an app, but mobile-friendly would be a big plus.
